envision/web/src/components/header.js

```diff
@@ -60,12 +60,27 @@ export default function Header_({
     setEgoView(egoView_);
   }
 
+  function displaySimName(id) {
```

Comment:
If you are worried about user supplied string, since the input comes from a user, it might be best to sanitise the input here.

Reply:
I am not sure a super long string can still be a valid key value on the client side?

Reply:
Alright!